guide/how-it-works/silent-payments.md

```diff
@@ -0,0 +1,360 @@
+
+{% include image-gallery.html pages = page.images_send %}
+
+
+With labels helping so much to improve coin selection, UIs might come up with better ways to choose (automatically or by users, manually) the coins to be used in a transaction. Note that the coins chosen for a particular transaction impact the derived on-chain address and user flows should be designed based on that.
+
+### Test transactions
+
+Test transactions are another way to alleviate user anxiety for sending on-chain payments, especially with static addresses, since the on-chain address is always visibly different from the static address.
+
+After the first use, users are typically on their own to use the application as it fits their needs, and everything should be set up appropriately. Features that are typically only relevant at later stages of the [usage life cycle]({{ '/guide/designing-products/usage-life-cycle/#first-use' | relative_url }}) may benefit from their own onboarding flows. Key is to present introductory information at the time it becomes relevant to the user.
+
+{% include image-gallery.html pages = page.images_test-txns %}
+
+## Receiving & scanning
+
+Since static addresses can now be used to pay a receiver several times, receiving flows are likely to be used less often, rarely even, since static addresses can be posted in public and others can simply get it from there. So whenever the receiver is sharing a payment request, users have the opportunity to associate valuable information to it – through labels and contacts; applications should facilitate this process..
+
+Some places where people can share static addresses (and add labels) to include:
+- Social media pages
+- Donate or fundraising websites
+- Exchanges
+- Email signatures
+
+<div class="center" markdown="1">
+
+{% include picture.html
+   modalImage = "/assets/images/guide/how-it-works/silent-payments/adding-labels-to-static-address@2x.png"
+   width = 250
+   height = 522
+   alt-text = "An application screen showing labels being added to a static address."
+   caption = "."
+   layout = "float-right-desktop -background -shadow"
+%}
+
+Regularly [backing up wallet]({{ '/guide/daily-spending-wallet/backup-and-recovery/landing-page/' | relative_url }}) and user data in a secure manner is essential for bitcoin applications. For small amounts of funds, [encrypted cloud backups]({{ '/guide/daily-spending-wallet/backup-and-recovery/cloud-backup/' | relative_url }}) are a convenient choice with a good tradeoff between security and convenience.
```

Comment:
Should this go into the backup section further down?

Reply:
Was a leftover from earlier, but I added a link to reference the cloud backup page already existing in the reference design in https://github.com/BitcoinDesign/Guide/pull/1109/commits/1b255e6d648679b326858aac40e44a26045e45b0 commit.